packages/style-engine/src/styles/background/index.ts

```diff
@@ -21,20 +27,12 @@ const backgroundImage = {
 			];
 		}
 
-		/*
-		 * If the background image is a string, it could already contain a url() function,
-		 * or have a linear-gradient value.
-		 */
-		if ( typeof _backgroundImage === 'string' ) {
-			return generateRule(
-				style,
-				options,
-				[ 'background', 'backgroundImage' ],
-				'backgroundImage'
-			);
-		}
-
-		return [];
+		return generateRule(
```

Comment:
This will return the default value instead of `[]`. 

Optional, but it makes it consistent with other style engine methods.

The consequence is that, if the input value cannot be processed, it'll be returned and not wiped from the face of the app.